### DISCUSSION TITLE
Prevent duplicate generated migration field

### DISCUSSION THREAD

**Discussion by fbuys** [luckyframework/lucky#1628] [Feature requests & Ideas] [answered]
It is possible to generate a resource with duplicate fields.
I think it makes sense to prevent this from happening and rather warn that a duplicate field is present?

Right now you can generate a resource with two titles like this:
`lucky gen.resource.browser Project title:String title:String`

I cannot think of a scenario where you would want two duplicate fields, I don't think this will work in Postgres either.

I am happy to work on a PR if this feature sounds reasonable?

**Comment by jwoertink** [comment, marked as answer]
That's a great catch! We should definitely prevent that.
